deploy/charts/monitoring/values.yaml

```diff
@@ -1,4 +1,4 @@
-grafanaImage: grafana/grafana:7.5.15
```

Comment:
Why this version and not 11.x?

Reply:
That is the first major version where I don't see the typing bug occurring. It should have been fixed in v9, but I still experienced weird (however different than in version 7) problems.